### PR TITLE
Fixes active tab

### PR DESCRIPTION
**Pull request by gutierri**
Fixs #62 

anim

### CODE REVIEW

File: browser.css [hunk 1]
```diff
@@ -64,6 +64,15 @@ img.larrybird {
 /* remove bottom border from active navbar tab */
 #react-root ._1Qfdd7IP {
 	box-shadow: none !important;
+	color: #1da1f2 !important;
+}
+
```
Comment (paulmolluzzo) [inline browser.css:70]: Can you please add a comment in here so we know why we need to add this? Thanks.

File: browser.css [hunk 2]
```diff
@@ -64,6 +64,15 @@ img.larrybird {
...
+#react-root ._1GdFzFv1:focus {
+	color: #8899a6;
+}
+
```
Comment (paulmolluzzo) [inline browser.css:75]: Same as https://github.com/sindresorhus/anatine/pull/77/commits/b27507f308050a92fd3e83905668fc1f9060d73d#r65113299

**Comment by paulmolluzzo** [comment]
Can confirm this is fixing the issue. I'd like to see the comment added to the CSS is all.

**Comment by sindresorhus** [comment]
Yes, would be nice if you could elaborate on how this fixes it ;)

**Comment by gutierri** [comment]
@sindresorhus @paulmolluzzo (ref 766d615a) That's good?

**Comment by paulmolluzzo** [comment]
@gutierri you might want to reference the original issue. Something like 

`Set color for navbar item on :focus to fix issue #62` and  `Set color for navbar item when :active to fix issue #62`

**Comment by gutierri** [comment]
@paulmolluzzo This seems to be better. I'll make the changes! :)

**Comment by paulmolluzzo** [comment]
Nice, thanks @gutierri!

**Comment by gutierri** [comment]
```
                                                                                  :)                                                                                                                                                                                                                                                                                                                                         
```

**Pull request merged**